src/activeList.js

```diff
@@ -0,0 +1,153 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable no-unused-vars */
+/* eslint-disable func-names */
+/* eslint-disable no-restricted-globals */
+/* eslint-disable import/no-mutable-exports */
+
+import _ from 'lodash';
+
+/* eslint-disable no-const-assign */
+const form = document.querySelector('#form');
+const inputF = document.querySelector('.inputF');
+const parentNode = document.querySelector('.parent-node');
+const enterErr = document.querySelector('.enter-err');
+const edit = document.querySelector('.edit');
+const enterBtn = document.querySelector('.btn');
+const removeAll = document.querySelector('.removeAll');
+
+let itemsArr = [];
+
+const removeItem = function (item) {
+  const removeIndex = itemsArr.indexOf(item);
+  itemsArr.splice(removeIndex, 1);
+};
+
+const updateItem = function (currentItemIndex, value) {
+  const newItem = itemsArr[currentItemIndex];
+  newItem.description = value;
+  itemsArr.splice(currentItemIndex, 1, newItem);
+  // eslint-disable-next-line no-use-before-define
+  saveLocalStorage(itemsArr);
+};
+
+// eslint-disable-next-line func-names
+const handleItem = function (itemData) {
+  const items = document.querySelectorAll('.show-list');
+  items.forEach((item) => {
+    // eslint-disable-next-line eqeqeq
+    if (item.querySelector('.tast').getAttribute('data-time') == itemData.index) {
+      item.querySelector('[data-done]').addEventListener('change', (e) => {
+        e.preventDefault();
+        const itemIndex = itemsArr.indexOf(itemData);
+        const currentItem = itemsArr[itemIndex];
+        // eslint-disable-next-line no-unneeded-ternary
+        currentItem.completed = currentItem.completed ? false : true;
+        itemsArr.splice(itemIndex, 1, currentItem);
+        // eslint-disable-next-line eqeqeq
+        if (currentItem.completed == false) {
+          item.querySelector('.tast').style.textDecoration = '';
+          item.querySelector('.drag-drop').style.display = 'flex';
+          item.querySelector('.trash').style.display = 'none';
+        } else {
+          item.querySelector('.tast').style.textDecoration = 'line-through';
+          item.querySelector('.drag-drop').style.display = 'none';
+          item.querySelector('.trash').style.display = 'flex';
+        }
+        // eslint-disable-next-line consistent-return
+        removeAll.addEventListener('click', (e) => {
+          e.preventDefault();
+          // eslint-disable-next-line eqeqeq
+          if (currentItem.completed == true) {
```

Comment:
I think it's better to use `=== (Triple equals operator)` here

```suggestion
          if (currentItem.completed === true) {
```